charts/harvester-cloud-provider/templates/rbac.yaml

```diff
@@ -15,11 +15,14 @@ rules:
     resources: [ "services", "nodes", "events" ]
     verbs: [ "get", "watch", "list", "update", "create", "patch" ]
   - apiGroups: [ "" ]
-    resources: [ "nodes/status", "services/status"]
+    resources: [ "nodes/status", "services/status" ]
     verbs: [ "update", "patch" ]
   - apiGroups: [ "coordination.k8s.io" ]
     resources: [ "leases" ]
     verbs: [ "get", "update", "create" ]
+  - apiGroups: [ 'policy' ]
+    resources: [ 'podsecuritypolicies' ]
+    verbs: [ 'use' ]
```

Comment:
currently, you are granting permissions for the pod to use **any** pod security policy defined across **all** namespaces (since it is given that cluster role). 

Instead, you should be specifying a specific resource name tied to a specific PSP that you deploy **for the Pods tied to this ClusterRole**. And ideally, that should be a Role bounded to the ServiceAccount via a RoleBinding so that it is only tied to your namespace.

See https://github.com/rancher/charts/blob/release-v2.6/charts/longhorn/longhorn/100.0.0%2Bup1.1.2/templates/psp.yaml#L1-L66 for an example of how Longhorn defines a PSP for its deployment.

Reply:
It's not the main task to add the PSP. We will add it in the next release.